.github/scripts/deny-git-deps.py

```diff
@@ -11,10 +11,8 @@
 
 from cargo_workspace import Workspace, DependencyLocation
 
+# Dependencies and the crate using them that are known to be bad.
 KNOWN_BAD_GIT_DEPS = {
-	'simple-mermaid': ['xcm-docs'],
-	# Fix in <https://github.com/paritytech/polkadot-sdk/issues/2922>
-	'bandersnatch_vrfs': ['sp-core'],
 	'subwasmlib': ['polkadot-zombienet-sdk-tests'],
```

Comment:
Why is this one allowed to stay?

Reply:
`polkadot-zombienet-sdk-tests` is not published, so it should not cause issues. Ideally `subwasmlib` should still be on crates-io. But that is outside of our control.

Reply:
Yes sure. Just talking about long to mid term plans. Can't we just fork and re-publush `subwasmlib` to bring it under our control?